_includes/sponsor.html

```diff
@@ -5,7 +5,14 @@
         height: 100%;
         margin-bottom: 40px;
     }
-
+    @media screen and (max-width: 736px) {
```

Comment:
@MBrouns Can you explain what you are doing here?

Reply:
Of course! The @media rule allows us to specify rules only for specific devices. Screen means everything with a screen - so no printers or screen readers. We then select only screens with leas then 736 horizontal pixels and apply the css below it only in those instances

Reply:
Awesome, did not know this. You add the `@media screen` to be sure that a `max-width` can be found? When there is no screen there is also no `max-width`.

Reply:
I don't have much experience designing for screenreaders so I'm not sure how that works